pages/documentation/developer/tutorials/network_modifications_groovy.md

```diff
@@ -0,0 +1,95 @@
+| InjectionPositionOrder | /                   | Required in Node/Breaker. The order position of the injection to create the corresponding extension. <br/> The injection will be positioned accordingly on the busbar section.    |
+| InjectionFeederName    | Id of the injection | Only in Node/Breaker. An optional name to be put in the ConnectablePosition extension and that will be displayed on the diagrams.                                                 |
+| InjectionDirection     | BOTTOM              | Only in Node/Breaker. An optional direction to be put in the ConnectablePosition extension and that will correspond to the direction of the injection in the Single line diagram. |
+
+
+# CreateLineBay.groovy
+
+In this script, the same node/breaker network is loaded and a line is created between two voltage levels S1VL2 and S2VL1.
+
+Before adding the line, the two voltage levels are:
+
+![](img/network_modifications_groovy/nb_network_s1vl2.png)
+![](img/network_modifications_groovy/nb_network_s2vl1.png)
+
+The script is adding the line between the generator GH3 (order position 50) and the load LD2 (order position 60) on S1VL2 and on the right of the line LINE_S2S3 (order position 30) on S2VL1.
+The order position for the new line in the script are taken to be between 50 and 60 on S1VL2 and higher than 30 on S2VL1.
+On S1VL2, the new line is pointing to the bottom, which is specified in the modification via the parameter ConnectablePosition.BOTTOM.
+On S2VL1, the direction of the new line is TOP.
+
+The voltage levels will then look like that after adding the line:
+
+![](img/network_modifications_groovy/nb_network_with_new_line_S1VL2.png)
+![](img/network_modifications_groovy/nb_network_with_new_line_s2vl1.png)
+
+The same method can be used to create a two-windings transformer on a network.
+
+Here are the options that must/can be filled:
+
+| Parameter             | Default value    | Description                                                                                                                                                                                                               |
+|-----------------------|------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| BranchAdder           | /                | The adder corresponding to the branch to be added. <br/>Should be created beforehand and contain all the required information.                                                                                            |
+| BusOrBusbarSectionId1 | /                | The ID of the busbar (node/breaker network) or of the bus (bus/breaker network) on which the branch should be connected on side 1.                                                                                        |
```

Comment:
Same remark as above, it may be useful to specify that it is a mandatory parameter.